Source/src/modules/ModuleProgram.h

```diff
@@ -129,6 +148,9 @@ namespace Hachiko
         Program* deferred_geometry_program = nullptr;
         Program* deferred_lighting_program = nullptr;
         Program* skybox_program = nullptr;
+        Program* diffuseIBL_program = nullptr;
+        Program* prefilteredIBL_program = nullptr;
+        Program* environmentBRDF_program = nullptr;
```

Comment:
These should be renamed like:
```suggestion
        Program* diffuse_ibl_program = nullptr;
        Program* prefiltered_ibl_program = nullptr;
        Program* environment_brdf_program = nullptr;
```